modules/monitoring/manifests/network_checks.pp

```diff
@@ -0,0 +1,22 @@
+# == Class: monitoring::network_checks
+#
+# Sets up vDC gateways as separate hosts so they can become parents of child
+# hosts dependent on the state of their gateway.
+#
```

Comment:
The description here mentions vDC gateways but the name of the defined type is much more general. Perhaps this could be renamed `monitoring::vpn_gateway` (singular) to make its intent clearer?